core/src/main/java/pt/webdetails/cda/endpoints/RestEndpoint.java

```diff
@@ -35,131 +32,167 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.DefaultValue;
 import javax.ws.rs.QueryParam;
-import javax.ws.rs.core.Context;
-import javax.servlet.http.HttpServletResponse;
-import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.Response;
 
 import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang.StringUtils;
 
 import pt.webdetails.cda.CdaCoreService;
 import pt.webdetails.cda.exporter.ExportOptions;
 import pt.webdetails.cda.exporter.ExportedQueryResult;
+import pt.webdetails.cda.exporter.ExporterException;
+import pt.webdetails.cda.exporter.UnsupportedExporterException;
 import pt.webdetails.cda.utils.DoQueryParameters;
 import pt.webdetails.cpf.PluginEnvironment;
-import pt.webdetails.cpf.Util;
 import pt.webdetails.cpf.repository.api.IReadAccess;
 import pt.webdetails.cpf.utils.CharsetHelper;
 import pt.webdetails.cpf.utils.MimeTypes;
 
-@Path( "/cda/api/utils" )
 /**
  * @deprecated
```

Comment:
Should we review the reasoning behind the `@deprecated` tag? Why was it deprecated and is it still, as we are still fixing things and "building" things on top? @graimundo

Reply:
@tgf is currently PTO but I'm guessing its marked as deprecated because the only class that was being used in practice was the pentaho implementation one. IMO using these endpoints in Foundry actually makes them as not deprecated. Are the `getCssResource` and `getJsResource` endpoints being used at all? Those are the ones that strike me as possible candidates for deprecation.